tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
Yes thats the issue I linked in my other comment. Read broke in 2.6 and is no fix on master (but there is still another issue with `printf` on the latest master that wasnt there previously [starting a format string with "--" cause printf to try to use the full string as an argument now]).

This scripts tends to show me quite well if something "breaks" with the read & printf builtin.